code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -162,19 +162,32 @@
 	color = "#669900" // rgb: 102, 153, 0
 	toxpwr = 0.5
 	taste_description = "death"
+	var/ingested = FALSE
 
 /datum/reagent/toxin/zombiepowder/on_mob_metabolize(mob/living/L)
 	..()
-	L.fakedeath(type)
+	ADD_TRAIT(L, TRAIT_FAKEDEATH, type)
 
 /datum/reagent/toxin/zombiepowder/on_mob_end_metabolize(mob/living/L)
 	L.cure_fakedeath(type)
 	..()
 
-/datum/reagent/toxin/zombiepowder/on_mob_life(mob/living/carbon/M)
-	M.adjustOxyLoss(0.5*REM, 0)
+/datum/reagent/toxin/zombiepowder/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
+	L.adjustOxyLoss(0.5*REM, 0)
+	if(method == INGEST)
+		ingested = TRUE //Created Var for ZP
+		L.fakedeath(type)
+
+/datum/reagent/toxin/zombiepowder/on_mob_life(mob/living/M)
 	..()
-	. = 1
+	if(ingested)
+		return TRUE
+	switch(current_cycle)
+		if(1 to 5)
+			M.confused += 1
+			M.drowsyness += 1
+		if(5 to INFINITY)
+			M.fakedeath(type)
```

Comment:
if you set ingested to TRUE here you wouldn't have to constantly handle the switch (although you should probably rename the var to like `fakedeath_active` or something).

5 is still incredibly low though, If you insist on having it down a person why not have an intermediate knockdown effect or stamina damage /w say slurring. That's more interesting and open to counter besides "lol better already be injected with calomel".

Reply:
It's just 5 less cycles then Chloral Hydrate and it still only produces 2u here per 5u of morphine, carpo, and copper.

Reply:
5 less cycles is a pretty big deal, the bonus of ZP should come from the fakedeath and the plausible deniability associated with that (oh romeo my romeo), not really the hardstun. That aspect should be the reward of doing more difficult transfer methods via the ingest.